resources/views/home.blade.php

```diff
@@ -1,23 +1,14 @@
 @extends('layouts.app')
 
 @section('content')
-<div class="container">
-    <div class="row justify-content-center">
-        <div class="col-md-8">
-            <div class="card">
-                <div class="card-header">Dashboard</div>
-
-                <div class="card-body">
-                    @if (session('status'))
-                        <div class="alert alert-success" role="alert">
-                            {{ session('status') }}
-                        </div>
-                    @endif
 
-                    You are logged in!
-                </div>
-            </div>
-        </div>
-    </div>
+<div class="container">
+    @if (auth()->user()->is_admin) {{-- admin user--}}
+    {
+    @include('admin/users')
+    }@else{ {{-- normal user --}}
+    @include('normal_users/user')
+    }
+    @endif
```

Comment:
I think it would be cleaner this way :)
```suggestion
    @if (auth()->user()->is_admin) {{-- admin user--}}
        @include('admin/users')
    @else {{-- normal user --}}
        @include('normal_users/user')
    @endif
```

Reply:
I removed "{" and "}".
But indent is fixed automatically by VSCode's Extension😔